src/main/java/net/fabricmc/tinyremapper/MetaInfFixer.java

```diff
@@ -39,7 +39,7 @@ public void transform(Path destinationDirectory, Path relativePath, InputStream
 			Path outputDir = outputFile.getParent();
 			if (outputDir != null) Files.createDirectories(outputDir);
 
-			try (OutputStream os = new BufferedOutputStream(Files.newOutputStream(outputFile))) {
+			try (OutputStream os = new BufferedOutputStream(Files.newOutputStream(outputFile, StandardOpenOption.CREATE))) {
```

Comment:
The default options (having passed none) is `StandardOpenOption.CREATE, StandardOpenOption.TRUNCATE_EXISTING, StandardOpenOption.WRITE`, is this for something like avoiding overwriting service files?

Reply:
I think that's how it was originally, maybe it was a bad merge